### ISSUE TITLE
Add comment to DB schema builder

### ISSUE THREAD

**Issue by nkostadinov** [yiisoft/yii2#9800]
It would be really nice if the migration script offered a simple comment method to add comments to field definitions e.g.

````
       $this->createTable('{{%customer}}', [
            'id' => $this->primaryKey(),
            'name' => $this->string()->comment("this is a comment"),
            ......
       ]);
````

**Comment by samdark** [comment]
Related to #https://github.com/yiisoft/yii2/issues/8148

**Comment by samdark** [comment]
#9441

**Comment by SilverFire** [comment]
Implemented in fde8aa6bcbb902aaea10f02f07fa87e11e5d2d17
